Grasshopper.fs

```diff
@@ -84,7 +89,7 @@ module Types =
         | /// <summary>
           ///     The given microcode command cannot be expressed in Grasshopper.
           /// </summary>
-          CommandNotImplemented of cmd : Microcode<CTyped<MarkedVar>, Sym<MarkedVar>> list
+          CommandNotImplemented of cmd : Microcode<CTyped<MarkedVar>, Sym<MarkedVar>>
```

Comment:
As above, things that used to be ||-composed lists of microcode are now just flattened into single microcode instructions